src/vs/workbench/contrib/relauncher/browser/relauncher.contribution.ts

```diff
@@ -258,6 +268,64 @@ export class WorkspaceChangeExtHostRelauncher extends Disposable implements IWor
 	}
 }
 
+export class RuntimeArgumentsChangeRelauncher extends Disposable implements IWorkbenchContribution {
+
+	private readonly disableChromiumSandbox = new ChangeObserver('boolean');
+	private readonly disableHardwareAcceleration = new ChangeObserver('boolean');
+	private readonly enableCrashReporter = new ChangeObserver('boolean');
+	private readonly locale = new ChangeObserver('string');
+
+	constructor(
+		@IHostService private readonly hostService: IHostService,
+		@IFileService private readonly fileService: IFileService,
+		@IEnvironmentService private readonly environmentService: IEnvironmentService,
+		@IProductService private readonly productService: IProductService,
+		@IDialogService private readonly dialogService: IDialogService
+	) {
+		super();
+
+		// Set default values.
+		this.disableChromiumSandbox.reset(false);
+		this.disableHardwareAcceleration.reset(false);
+		this.enableCrashReporter.reset(true);
+		this.locale.reset('en');
+
```

Comment:
What if the user has already configured some of the values? We cannot just reset to defaults, we actualy need to read the file to know their values?

Reply:
We only need a restart when a user changes the value after launching the instance, which would be covered by comparing with the defaults right ?